vue/components/ui/organisms/details-expandable/details-expandable.vue

```diff
@@ -0,0 +1,178 @@
+                    />
+                </slot>
+                <div class="details-expandable-value">
+                    <slot v-bind:name="sectionName + '-' + name" v-bind:field-value="value">
+                        <input-ripe v-bind:value="value" />
+                    </slot>
+                </div>
+            </div>
+        </div>
+        <div class="details-expandable-btn-container">
+            <button-color
+                class="details-expandable-btn-discard"
+                v-bind:size="'small'"
+                v-bind:text="discardButtonText"
+                v-bind:secondary="true"
+                v-bind:icon="'close'"
+                v-on:click="onDiscardHandler"
+            />
+            <button-color
+                class="details-expandable-btn-save"
+                v-bind:size="'small'"
+                v-bind:text="saveButtonText"
+                v-bind:icon="'save'"
+                v-on:click="onSaveHandler"
+            />
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+.details-expandable {
+    padding: 0px 16px 0px 16px;
+}
+
+.details-expandable-caret {
+    height: 24px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 24px;
+}
+
+.details-expandable-btn-container {
+    float: right;
+    height: 25px;
+    margin-top: 50px;
+}
+
+.details-expandable-btn-discard {
+    margin-right: 6px;
+    min-width: 125px;
+}
+
+.button-color.button-color-small {
+    min-width: 125px;
+}
+
+.details-expandable-section {
+    cursor: pointer;
+    display: flex;
+}
+
+.details-expandable-section-hide {
+    border-bottom: 1px solid #e4e8f0;
+    height: 25px;
+    margin-top: 25px;
+    overflow: hidden;
+    padding-bottom: 25px;
+    position: relative;
+}
+
+.details-expandable-section-show {
+    height: auto;
+    margin-top: 25px;
+    overflow: visible;
+    padding-bottom: 25px;
+    position: relative;
+}
+
+.details-expandable-header {
+    color: #0d0d0d;
+    font-size: 12px;
+    font-weight: 700;
+    letter-spacing: 0.6px;
+    text-transform: uppercase;
+}
+
+.details-expandable-title {
+    color: #57626e;
+}
+
+.details-expandable-value {
+    margin-top: 10px;
+}
+
+.details-expandable-row {
+    margin-top: 25px;
+}
+</style>
+<script>
+export const DetailsExpandable = {
+    name: "details-expandable-ripe",
+    props: {
+        data: {
+            type: Object,
+            required: true
+        },
+        labelFontSize: {
+            type: Number,
+            default: null
+        },
+        discardButtonText: {
+            type: String,
+            default: "Discard changes"
+        },
+        saveButtonText: {
+            type: String,
+            default: "Save changes"
+        }
+    },
+    data: function() {
+        const isExpanded = {};
+        Object.keys(this.data).forEach(k => {
+            isExpanded[k] = false;
+        });
+
+        return {
+            isExpanded

```

Comment:
Follow the standard of other components, be a little more verbose when declaring data fields.
```suggestion
            isExpanded: isExpanded

```